pycaching/cache.py

```diff
@@ -153,8 +153,10 @@ def _from_print_page(cls, geocaching, guid, soup):
         cache_info["attributes"] = {attr_name: attr_setting == "yes" for attr_name, _, attr_setting in attributes}
         if "attribute" in cache_info["attributes"]:  # 'blank' attribute
             del cache_info["attributes"]["attribute"]
-        cache_info["summary"] = content.find("h2", string="Short Description").find_next("div").text
-        cache_info["description"] = content.find("h2", string="Long Description").find_next("div").text
+        cache_info["summary"] = content.find("h2", string="Short ").find_next("div").text
```

Comment:
Why did you change this string from "Short Description" to "Short "? This seems to be wrong and lets the existing tests fail.

Reply:
I didn’t. That’s so weird :/